packages/core/src/connectivity/scp-cf/xsuaa-service-xssec.ts

```diff
@@ -0,0 +1,85 @@
+import * as xssec from '@sap/xssec';
```

Comment:
Why is this file named `...-xssec` - seems not very speaking to me. I would expect this to be in the `xsuaa-service.ts`.

Reply:
Discussed, we can rename this as `xsuaa-service.ts` and rename the old `xsuaa-service.ts` to `...-legacy.ts`